docs/source/packages/core-implementations.rst

```diff
@@ -40,7 +40,7 @@ Core Implementations
 
     - :ref:`Effect <module-daml-finance-lifecycle-effect-1975>`: A contract encoding the *consequences of a lifecycle event* for one unit of the target instrument
     - :ref:`ElectionEffect <module-daml-finance-lifecycle-electioneffect-99924>`: A contract encoding the *consequences of an election* for one unit of the target instrument
-    - :ref:`DateClock <module-daml-finance-lifecycle-dateclock-31311>`: Rule contract that allows an actor to process/claim effects, returning settlement instructions
```

Comment:
If I'm allowed to revive https://github.com/digital-asset/daml-finance/issues/100 @georg-da @brianweir-da, I think that the `DateClock` should not live in `Lifecycle` but rather in `Data` :)